tests/integration/src/tests/smoke.rs

```diff
@@ -0,0 +1,34 @@
+#[allow(unused_imports)]
+use fluvio_command::CommandExt;
+
+#[fluvio_test_macro::fluvio_test(topic = "smoke")]
+fn smoke_test() {
+    // Magic variable
+    println!("TestOption: {:?}", &test_option);
```

Comment:
Are you planning on getting rid of the magic variable? I don't think it should be included in the final thing

Reply:
Yes. I have that println!() there as on-going debugging / a reminder to get rid of the variable. Providing the test writer with the test context for their own test case setup will not be provided through magic variables. (Or at least, it won't have the appearance of coming from nowhere).

Sehyo gave me an example that I'll be trying out. Using a function signature like:

```rust
#[fluvio_test_macro::fluvio_test(topic = "smoke")]
async fn smoke_test(test_opt: IntegrationTestOptionX) { ... }
```

Where `test_opt` can be the name of the var that I end up creating with the macro. The macro args and initialized cluster info available in a more obvious way.